### PR TITLE
Matrix_identical in java

### PR DESCRIPTION
**Pull request by raksha009**
issue no #1628 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Addition of new code
- 

#### Languages Used:
- Java
- 

#### Files Added:
- Matrix_identical.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by raksha009** [comment]
plz review my PR @singh-shreya6 @asha15 @raghav-dalmia

File: Matrix_Operations/Java/Matrix_identical.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        if (columns_first != columns_second || rows_first != rows_second) {
+            System.out.println(" Matrices's order is not identical ");
+        }
+        else {
+
+            System.out.println(" Enter elements of 1st matrix ");
+            for (int i = 0; i < rows_first; i++) {
+                for (int j = 0; j< columns_first; j++) {
+                    matrix1[i][j] = s.nextInt();
```
Comment (singh-shreya6) [inline Matrix_Operations/Java/Matrix_identical.java:46]: Space before <

File: Matrix_Operations/Java/Matrix_identical.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ Enter number of columns 
+2
+ Enter elements of 1st matrix 
+1 2
+0 4
+ Enter elements of 2nd matrix 
+1 2
+7 6
+
+OUTPUT
```
Comment (singh-shreya6) [inline Matrix_Operations/Java/Matrix_identical.java:83]: Please add one more sample input/output for identical scenario

**Comment by raksha009** [comment]
Done @singh-shreya6

**Comment by singh-shreya6** [comment]
Please squash your commits @raksha009

**Comment by singh-shreya6** [comment]
@raksha009 Please squash?

**Comment by raksha009** [comment]
Done the changes @singh-shreya6 
Sorry for delay, was having my continuous online exams for past few days.

**Comment by singh-shreya6** [comment]
It is okay @raksha009

**Comment by raksha009** [comment]
Thanks :)

**Pull request merged**